optuna/structs.py

```diff
@@ -128,6 +175,52 @@ def _validate(self):
                     "The value {} of parameter '{}' isn't contained in the distribution {}.".
                     format(param_value, param_name, distribution))
 
+    @property
+    def distributions(self):
+        # type: () -> Dict[str, BaseDistribution]
+        """Return the distributions for this trial.
+
+        Returns:
+            The distributions.
+        """
+
+        return self._distributions
+
+    @distributions.setter
+    def distributions(self, value):
+        # type: (Dict[str, BaseDistribution]) -> None
+        """Sets the distributions for this trial.
```

Comment:
By convention, the Optuna document does not use the third person singular. 
```suggestion
        """Set the distributions for this trial.
```

Reply:
Thanks for letting me know. Got it.